docs/layouts/shortcodes/generated/flink_connector_configuration.html

```diff
@@ -236,5 +236,11 @@
             <td>Integer</td>
             <td>Defines a custom parallelism for the unaware-bucket table compaction job. By default, if this option is not defined, the planner will derive the parallelism for each statement individually by also considering the global configuration.</td>
         </tr>
+        <tr>
+            <td><h5>watermark.in.batch.mode</h5></td>
```

Comment:
Can we just introduce a endInput watermark?

Reply:
done